### ISSUE TITLE
this container is not working

### ISSUE THREAD

**Issue by 7eptem** [vexorian/dizquetv#477]
i installed both on two separate machines and the container is not working, changing the port does nothing

**Comment by hendrix04** [comment]
Yes, it always starts on port 8000 no matter what and the container does nothing to change how it exposes the port.

I installed from CA on Unraid and this might be more of an unraid template issue.
